internal/gmm/handler.go

```diff
@@ -2164,8 +2168,7 @@ func HandleAuthenticationFailure(ue *context.AmfUe, anType models.AccessType,
 			gmm_message.SendAuthenticationRequest(ue.RanUe[anType])
 		}
 	} else if ue.AuthenticationCtx.AuthType == models.AuthType_EAP_AKA_PRIME {
-		switch cause5GMM {
-		case nasMessage.Cause5GMMngKSIAlreadyInUse:
```

Comment:
I think we should keep use `switch`

Reply:
Should this part have no linter?

Reply:
Add some other `Case`, leave code empty, and TODO comment. 
I think this would pass Linter.